bin/gobblin-mapreduce.sh

```diff
@@ -18,6 +18,7 @@ function print_usage(){
   echo "  --jt <job tracker / resource manager URL>      Job submission URL: if not set, taken from \${HADOOP_HOME}/conf"
   echo "  --fs <file system URL>                         Target file system: if not set, taken from \${HADOOP_HOME}/conf"
   echo "  --jars <comma-separated list of job jars>      Job jar(s): if not set, \"$FWDIR_LIB\" is examined"
+  echo "  --conf <job conf file>                         Job configuration file: if not set, taken from \${JOB_CONFIG_FILE}"
```

Comment:
`--conf` doesn't seem to be optional, according to line 81.
If it is, then line 16 should be revised.